script.js

```diff
@@ -0,0 +1,16 @@
+// Select Form
```

Comment:
You got off to a great start here! Sometimes using pseudocode and commenting out all my steps first before I start helps keep me organized. For example, namespacing an app object and using a document ready with an init method can help you along in calling functions on events, etc.